test/scenarios-periodics/el92-src@cncf-conformance.sh

```diff
@@ -106,6 +162,10 @@ scenario_remove_vms() {
 }
 
 scenario_run_tests() {
-    prepare_hosts
-    run_sonobuoy
-}
\ No newline at end of file
+    start_junit
+    trap "close_junit" EXIT
```

Comment:
The framework creates and closes the file, so you don't want these 2 lines. Calling start_junit again will overwrite any existing content and having the trap will cause the file to be closed twice so the syntax is not correct.

Reply:
@dhellmann , it looks like the tests do not work without this change. 
The reason is because the junit is initialized in `run_tests` function, which is never called for CNCF. 
I have to return the code back

Reply:
This function is the plugin interface used by `run_tests`. If `run_tests` is not called, then these lines wouldn't do anything, right?

Reply:
Looking at `scenario.sh`, running tests is a separate action and it calls `action_run` [function](https://github.com/openshift/microshift/blob/main/test/bin/scenario.sh#L911). This function is different from `action_create` because it does not directly call `start_junit`.

The latter is only called inside `run_tests` function (RF invocation), which is never used in the CNCF test because it's not written using RF.

One possible solution to this problem would be to move the following block to `action_run` instead of `run_tests`.
```
    start_junit
    trap "close_junit; sos_report" EXIT
```

Then, it will always be called regardless of what's executed in the scenario run.

Reply:
I uploaded this implementation - let me know what you think.